packages/headless/src/controllers/field-suggestions/facet/headless-field-suggestions.ts

```diff
@@ -11,14 +63,136 @@ import {Subscribable} from '../../controller/headless-controller';
  *
  * This controller is a wrapper around the basic facet controller search functionality, and thus exposes similar options and properties.
  */
-export interface FieldSuggestions extends FacetSearch, Subscribable {}
+export interface FieldSuggestions extends Subscribable {
+  /**
+   * Performs a field suggestions search with a query.
+   *
+   * @param text - The query to search.
+   */
+  updateText(text: string): void;
 
-export interface FieldSuggestionsOptions extends FacetOptions {}
+  /**
+   * Shows more field suggestions for the current query.
+   */
+  showMoreResults(): void;
+
+  /**
+   * Performs a field suggestions search with the current query.
+   */
+  search(): void;
+
+  /**
+   * If a facet exists with the configured `facetId`, selects a facet value.
+   *
+   * @param value - The field suggestion for which to select the matching facet value.
+   */
+  select(value: FieldSuggestionsValue): void;
+
+  /**
+   * If a facet exists with the configured `facetId`, selects a facet value while deselecting other facet values.
+   *
+   * @param value - The field suggestion for which to select the matching facet value.
+   */
+  singleSelect(value: FieldSuggestionsValue): void;
+
+  /**
+   * Resets the query and empties the suggestions.
+   */
+  clear(): void;
+
+  /**
+   * Updates the field suggestions captions.
+   *
+   * @param captions - A dictionary that maps field values to field suggestion display names.
+   */
+  updateCaptions(captions: Record<string, string>): void;
+
+  state: FieldSuggestionsState;
+}
+
+// TODO: In v2, move these options into `FieldSuggestionsOptions` and remove `facetSearch`.
+export interface FieldSuggestionsFacetSearchOptions {
+  /**
+   * A dictionary that maps field values to field suggestion display names.
+   */
+  captions?: Record<string, string>;
+
+  /**
+   * The maximum number of suggestions to fetch.
+   *
+   * @defaultValue `10`
+   */
+  numberOfValues?: number;
+
+  /**
+   * The query to search the field suggestions with.
+   */
+  query?: string;
+}
+
+export interface FieldSuggestionsOptions {
+  /**
+   * The field for which you wish to get field suggestions.
+   */
+  field: string;
+
+  /**
+   * @deprecated This option has no effect.
+   */
+  delimitingCharacter?: string;
+
+  /**
+   * A unique identifier for the controller. By default, a random unique identifier is generated.
+   *
+   * If a facet shares the same id, then its values are going to be selectable with `select` and `singleSelect`. However, you must make sure to build the field suggestions controller after the facet's controller.
+   */
+  facetId?: string;
+
+  /**
+   * The options related to search.
+   */
+  facetSearch?: FieldSuggestionsFacetSearchOptions;
```

Comment:
One of the flaws of trying to make sure this is non-breaking, is that `facetSearch` needs to stay here, even though it makes no sense in the context of field suggestions.